Sources/KlaviyoCore/KlaviyoEnvironment.swift

```diff
@@ -111,6 +115,9 @@ public struct KlaviyoEnvironment {
     public var klaviyoAPI: KlaviyoAPI
     public var timer: (Double) -> AnyPublisher<Date, Never>
 
+    public var SDKName: String
+    public var SDKVersion: String
```

Comment:
How about changing to `sdkName` and `sdkVersion`, to be consistent with Swift naming conventions?